src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,63 @@
 
 import org.springframework.stereotype.Service;
 
+/**
+ * Service class providing basic calculator operations.
```

Comment:
5. **Dependency Injection**: Since you are using Spring's `@Service` annotation, consider injecting dependencies if needed in the future.